docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -33,11 +33,117 @@ Understanding GBD's Disease and Health Categories
+    - GBD calculates these very differently than sequelae despite them being clinically the same. 
+  * - Sequela or Sequelae 
+    - The medical conditions that can occur among people who contract a disease or suffer an injury. In other words, the negative health effects of a cause that are associated with certain health states. For example, chronic kidney disease can be a sequela of diabetes, neck pain can be a sequela of whiplash, and foot ulcers is a sequela of diabetes.
+    - "Sequela" is singular. "Sequelae" is plural. 
 
+
+Understanding the Definitions within the GBD Hierarchy Context
+--------------------------------------------------------------
+
+To understand the above definitions in practice, it is helpful to think of them 
+in the context of a specific example. We will use cardiovascular disease for this. 
+In particular, we will examine: 
+
+- Causes: ischemic heart disease (IHD) and hypertensive heart disease (HHD) 
+- Health states: medically managed, mild, moderate and severe heart failure 
+- Impairments: heart failure 
+- Sequelae: numerous, including "severe heart failure due to IHD" and "AMI first 2 days" among others 
+
+Causes
+++++++
+
+The cause hierarchy diagrams for IHD and HHD are included here as a reference. 
+
+
+**IHD**:
+
+.. image:: cause_hierarchy_ihd.svg
+
+**HHD**: 
+
+.. image:: cause_hierarchy_hhd.svg
+
+As you can see, the cause hierarchy slowly breaks down cause groups into more specific 
+diseases, moving from all non-communicable diseases to specific types of cardiovascular 
+disease. At each level, the cause hierarchy is mutually exclusive and 
+collectively exhaustive - meaning all disease and death can be placed into a group at 
+each level of the diagram. 
+
+Both IHD and HHD are within the more general cardiovascular disease cause (c_491). IHD and HHD 
+are causes within GBD. 
+
+Sequelae
+++++++++
+
+Once you have selected a specific cause - for example, IHD - you can then identify the 
+sequelae within that cause. These are all of the items listed below IHD and HHD in the 
+diagrams above. 
+
+Sequelae detail a specific medical condition associated with 
+a cause. For example, IHD has medical conditions like acute myocardial infarction 
+(a heart attack) and angina (chest pain) as associated sequelae. These are the things 
+a patient might be experiencing, or a doctor might identify. 
+
+Sequelae often contain a level of severity for that condition - for example, mild and moderate 
+angina are both sequelae within IHD. 
+
+Health States
++++++++++++++
+
+A health state is the amount of health or disability within a sequelae. This is summarized 
+through the disability weight. More information on this can be found in `this section of the cause model page <https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_model_components/causes/index.html#disability-weights>`_. 
```

Comment:
I think it would be worth noting that you can look up health states and their disability weights in the GBD methods appendix. 

Also, I'm not sure where we would want to handle this, but if we can have multiple health states for a given sequela (like the uterine fibroid example above), I think we should note how to determine the overall DW of that sequela.